src/AdminAan/SFA.DAS.AdminAan/Infrastructure/IAanHubRestApiClient.cs

```diff
@@ -28,6 +29,10 @@ public interface IAanHubRestApiClient
     [AllowAnyStatusCode]
     Task<Response<PostEventCommandResult>> PostCalendarEvents([Header(Constants.ApiHeaders.RequestedByMemberIdHeader)] Guid requestedByMemberId, [Body] PostEventCommand command, CancellationToken cancellationToken);
 
+    [Put("calendarEvents/{calendarEventId}")]
+    Task<Response<Unit>> PutCalendarEvent([Header(Constants.ApiHeaders.RequestedByMemberIdHeader)] Guid requestedByMemberId, [Path] Guid calendarEventId, [Body] PutEventCommand command, CancellationToken cancellationToken);
```

Comment:
This should be `Task PutCalendarEvent(....)` so there is no return type as we dont need to inspect the responce, we just let it fail here if it is not successful.

Reply:
yes